### DISCUSSION TITLE
第3节课作业(10班)

### DISCUSSION THREAD

**Discussion by isLinXu** [InternLM/Tutorial#264] [作业]
请在下方提交第 3 节课作业链接⬇️

**Comment by Tian-CASIA** [comment]
https://blog.csdn.net/qq_55090315/article/details/135491769?spm=1001.2014.3001.5501

**Comment by admlrz** [comment]
https://mp.csdn.net/mp_blog/creation/editor/135515369

**Comment by dearSocrates** [comment]
https://blog.csdn.net/rabbit9798/article/details/135465834?spm=1001.2014.3001.5501

**Comment by hannah803** [comment]
https://hannahliu.framer.website/blog/internlm-training-lec3

**Comment by jiangye520** [comment]
https://mp.weixin.qq.com/s?__biz=MzkxOTU4NzEyOQ==&mid=2247492661&idx=1&sn=43f5f6b311e6536cf1f0ef8af78f82a5&chksm=c19d7463f6eafd758f4fce76857a3fe642581786768281b85e1ff02bb32183c5cdc649e9df6d&token=199526352&lang=zh_CN#rd

**Comment by JokerOfRevenge** [comment]
https://www.processon.com/v/659f538930aabc3bad3c369f

**Comment by cemcke** [comment]
https://blog.csdn.net/qq_62600724/article/details/135512222?spm=1001.2014.3001.5501

**Comment by lishuang70345** [comment]
https://blog.csdn.net/weixin_42748604/article/details/135514789?spm=1001.2014.3001.5502

**Comment by zhangyutuan** [comment]
https://www.processon.com/view/link/65abe99b7fce4a25f8d9f65d

**Comment by adela778** [comment]
https://blog.csdn.net/Adela0546/article/details/135759788
